docs/develop/debug-office-add-ins-in-visual-studio.md

```diff
@@ -96,10 +96,28 @@ Next, Visual Studio does the following:
 
 ### Debug the Excel, PowerPoint, or Word add-in
 
-1. Launch the add-in in the Office application. For example, if it is a task pane add-in, it will have added a button to the **Home** ribbon (for example, a **Show Taskpane** button). Select the button in the ribbon. 
+The best method for debugging an add-in in Visual Studio 2022 depends on whether the add-in is running in WebView2, which is the webview control that is associated with Microsoft Edge (Chromium), or an older webview control. To determine which webview control is being used, see [Browsers used by Office Add-ins](../concepts/browsers-used-by-office-web-add-ins.md). If your computer is using WebView2, see [Use the built-in Visual Studio debugger](#use-the-built-in-visual-studio-debugger). For any other webview control, see [Use the browser developer tools](#use-the-browser-developer-tools).
+
+#### Use the built-in Visual Studio debugger
+
+1. Set breakpoints, as needed, in the source JavaScript or TypeScript files. You can do this either before or after you start the add-in as described in the earlier section [Start the Excel, PowerPoint, or Word add-in project](#start-the-excel-powerpoint-or-word-add-in-project).
```

Comment:
consider: uplevel and move the "Start the Excel, PowerPoint, or Word add-in project" before the debugging sections and rename to something like "Start the add-in project"

Reply:
Could you clarify? It's already before the debugging sections. Also, what is the advantage of the section name change? The article title sets the context as debugging.

Reply:
I was thinking "Start the Excel, PowerPoint, or Word add-in project" were used by WXPO so it would make sense to pull it out as an H2 before "Debug an Excel, PowerPoint, or Word add-in project". However, I realize that the instructions for starting WXP are different from starting OLK.